### PR TITLE
internationalized strings on maps

### LINKED ISSUE #574: internationalize strings on maps
Some strings are not internationalized for maps
- [x] All on the create map page
- [ ] popup on page when upload complete on create map page
- [x] all on calibration page
- [ ] on map page: the word "edited" added to first column when a change is made
- [ ] popup to confirm delete of a map
- [x] map button for map page and map button for graphic type on main page

### LINKED ISSUE #623: internationalize strings on meter page
### Describe the bug

The meter page has hard coded strings that are not internationalized as noted by @LukeLavan  in PR #622. These need to have strings defined in data.json so they show in appropriate language.

### Expected behavior

Strings on meter page would change to appropriate language.

### Screenshots

none

### Machine info (please complete the following information)

 NA

### Additional context

This is similar to issue #574 for maps and could be done at a similar time.

### PR DESCRIPTION
**Pull request by jina2k**
# Description

Fixes #574 , I have added all the strings mentioned in the issue.
Also Fixes #623 , for when editing meters. On the side I have also internationalized the managing users, and the map calibration page.

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

Current limitations are using the translate function, which is currently "deprecated". This only affects the map calibration page.

### CODE REVIEW

File: src/client/app/translations/data.json [hunk 1]
```diff
@@ -146,6 +151,8 @@
 		"map.new.submit": "Save and continue",
 		"map.note": "Note",
 		"map.upload.new.file": "Redo",
+		"map.load.complete": "Map load complete from",
```
Comment (huss) [inline src/client/app/translations/data.json]: Slightly wrong line for alphabetical.

File: docker-compose.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ services:
             - PGDATA=/var/lib/postgresql/data/pgdata
             - POSTGRES_PASSWORD=pleaseChange # default postgres password that can be changed for security.
         volumes:
-            - ./postgres-data:/var/lib/postgresql/data/pgdata
+            - ./postgres-data:/var/lib/postgresql/pgdata
```
Comment (huss) [inline docker-compose.yml]: Unsure why this was changed given the PGDATA defined above? Can you help me understand?

File: src/client/app/components/admin/UsersDetailComponent.tsx [hunk 3]
```diff
@@ -52,8 +53,8 @@ export default function UserDetailComponent(props: UserDisplayComponentProps) {
 						<thead>
 							<tr>
 								<th> Email </th>
-								<th> Role </th>
-								<th> Action </th>
+								<th> <FormattedMessage id='role'/> </th>
```
Comment (huss) [inline src/client/app/components/admin/UsersDetailComponent.tsx:56]: The translations already have "e-mail" so maybe do "Email" above.

File: src/client/app/components/admin/CreateUserComponent.tsx [hunk 4]
```diff
@@ -37,7 +38,7 @@ export default function CreateUserFormComponent(props: CreateUserFormProps) {
 
 	return (
 		<div className='container-fluid'>
-			<h1 style={titleStyle}> Create User </h1>
+			<h1 style={titleStyle}> <FormattedMessage id='create.user'/> </h1>
```
Comment (huss) [inline src/client/app/components/admin/CreateUserComponent.tsx:41]: A few lines down has "Email". Could it be translated with "e-mail" as long as doing this file?

**Review by huss — CHANGES_REQUESTED** [review]
This is good work and testing shows it works as expected. Just a few small suggestions and one question that would be nice to address.

File: docker-compose.yml (hunk 2 above)
Comment (jina2k) [reply docker-compose.yml]: mentioned this issue in #656 , in the meantime, reverted the change.

File: src/client/app/translations/data.json (hunk 1 above)
Comment (jina2k) [reply src/client/app/translations/data.json]: fixed this change, also moved a few other lines

File: src/client/app/components/admin/UsersDetailComponent.tsx (hunk 3 above)
Comment (jina2k) [reply src/client/app/components/admin/UsersDetailComponent.tsx:56]: Found an issue with e-mail only being available for Spanish in the data.json translation file, while french and english had email instead. Fixed it to have email for all 3, and included it in the following code.

File: src/client/app/components/admin/CreateUserComponent.tsx (hunk 4 above)
Comment (jina2k) [reply src/client/app/components/admin/CreateUserComponent.tsx:41]: As mentioned above, fixed the issue regarding email.

**Comment by jina2k** [comment]
For now, I have also included translations for calibrating the map. This uses the translate function, which is currently outdated due to getChildContext being deprecated. Later on, createIntl can be used instead to replace this deprecated function, as shown in the following link: https://stackoverflow.com/questions/59671390/typeerror-intlprovider-getchildcontext-is-not-a-function

**Review by huss — APPROVED** [review]

**Comment by huss** [comment]
Updates did all items in review and also cleaned up some other items in a nice way. Thanks for the good and quick work @jina2k . This PR is ready to merge.

**Pull request merged**